test/e2e/smoke-test/packages/local-path-storage/0.0.20/local-path-storage-test.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Checking package is installed or not
+tanzu package installed list | grep "local-path-storage.community.tanzu.vmware.com"
+exitcode=$?
+
+if [ "${exitcode}" == 1 ] 
+then
+  version=$(tanzu package available list local-path-storage.community.tanzu.vmware.com | tail -n 1 | awk '{print $2}')
+  tanzu package install local-path-storage --package-name local-path-storage.community.tanzu.vmware.com --version "${version}"
+fi
+
+MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+kubectl create -f "${MY_DIR}"/pvc.yaml
+kubectl create -f "${MY_DIR}"/pod.yaml
```

Comment:
We could use `kubectl apply` here to keep it declarative

Reply:
Also, we could run this test in a particular test namespace, say `test-${RANDOM}` namespace, so that it won't clash with any other possible resources with the same name in the default namespace that the user is on, which is popularly `default` namespace. But yeah, the tendency of clashes happening in resource names is rare but it's still possible. Not saying this is a critical thing, just an idea

Reply:
An alternative would be to use random names or suffixes for the resource names but then you would have to name each resource that's part of the test using a random name / suffix. Using a random namespace is easier with constant name for the resources